### ISSUE TITLE
Apply padding in Parent Layout

### ISSUE THREAD

**Issue by desmond1121** [Yalantis/Phoenix#24]
I've noticed that you handle bottom padding in scroll back. That would cause much time in "measure". 
I catched a systrace of your "scroll back":

[trace](https://github.com/desmond1121/Android-Ptr-Comparison/blob/master/traces/yalantis_back.PNG)

I understand why you need this logic, `View.offsetTopAndBottom` would destroy the bottom padding. But this not handled in scroll down yet. Actually, you can notify user to set padding in parent layout such as following code, and handle padding logic in `onLayout()` of `PullToRefreshView`.

```
<com.yalantis.phoenix.PullToRefreshView
        android:id="@+id/pull_to_refresh"
        android:padding="10dp"
        android:layout_width="match_parent"
        android:layout_height="match_parent">

    <ListView
        android:id="@+id/yalantis_list_view"
        android:divider="@null"
        android:dividerHeight="0dp"
        android:layout_width="match_parent"
        android:layout_height="match_parent" />
</com.yalantis.phoenix.PullToRefreshView>
```

**Comment by shliama** [comment]
Thanks for reporting the issue.
I'll look into it this week and try to find  a workaround because it would be hard to ask every library user to update their layouts :(

**Comment by desmond1121** [comment]
Yes, it is. You can just take padding property value by content's attributes, and apply it in your
it in your `onLayout()` :)
